apps/android/app/src/main/java/chat/simplex/app/SimplexApp.kt

```diff
@@ -35,9 +36,18 @@ external fun chatRecvMsgWait(ctrl: ChatCtrl, timeout: Int): String
 external fun chatParseMarkdown(str: String): String
 
 class SimplexApp: Application(), LifecycleEventObserver {
-  val chatController: ChatController by lazy {
-    val ctrl = chatInit(getFilesDirectory(applicationContext))
-    ChatController(ctrl, ntfManager, applicationContext, appPreferences)
+  lateinit var chatController: ChatController
+
+  fun initChatController(useKey: String? = null) {
+    val dbKey = useKey ?: DatabaseUtils.getDatabaseKey() ?: ""
+    val res = DatabaseUtils.migrateChatDatabase(dbKey)
+    val ctrl = chatInitKey(getFilesDirectory(applicationContext), dbKey)
```

Comment:
```suggestion
    val ctrl = chatInitKey(getFilesDirectory(applicationContext), "") // dbKey)
```

for now we can just pass an empty key